File: TilesAPI/GetTile.js
let data = null;
const xhr = new XMLHttpRequest();

const x = 31992;
const y = 20452;
const z = 16;
const token = "ADD YOUR TOKEN HERE";

const url = "https://tiles.hibirdi.com/zxy/" + z +
            "/" + x +
            "/" + y +
            "?token=" + token;

xhr.addEventListener("readystatechange", function () {
  if (this.readyState === 4) {
    console.log(this.responseText);
  }
});

xhr.open("GET", url);
xhr.send(data);

File: TilesAPI/GetTile.py
import requests

z = 16
x = 31992
y = 20452
token = "ADD YOUR TOKEN HERE"

url = ("https://tiles-api.hibirdi.com/tiles/zxy/{}/{}/{}/?token={}"
       .format(z, x, y, token))

response = requests.request("GET", url)

print(response.text)
